tailwind.config.ts

```diff
@@ -18,6 +18,9 @@ const config = {
       },
     },
     extend: {
+      lineHeight: {
+        'custom-58': '58.09px',
```

Comment:
Remove this! Follow what we have set up, the colors can be found on `localhost:3000/guides`  for you to see easily

Reply:
Also, attach screenshot and video of your component 

Also, screenshots of lint and pnpm tests